tests/test_pixelScaleBoundedField.py

```diff
@@ -0,0 +1,114 @@
+
+        self.bbox = lsst.geom.Box2I(lsst.geom.Point2I(0, 0), lsst.geom.Extent2I(200, 200))
+        self.points = [crpix, ]
+        # ensure that the points are all Point2D
+        self.points.extend(lsst.geom.Box2D(self.bbox).getCorners())
+
+        self.boundedField = PixelScaleBoundedField(self.bbox, self.skyWcs)
+
+    def testConstruct(self):
+        self.assertEqual(self.boundedField.getBBox(), self.bbox)
+        self.assertEqual(self.boundedField.getSkyWcs(), self.skyWcs)
+        self.assertEqual(self.boundedField.getDefaultInverseScale(),
+                         1.0 / self.skyWcs.getPixelScale().asDegrees()**2)
+
+    def _computeExpected(self, points):
+        """Return an array with the expected result of evaluate(point)."""
+        expect = np.zeros(len(points), dtype=float)
+        for i, point in enumerate(points):
+            scale = self.skyWcs.getPixelScale(lsst.geom.Point2D(point)).asDegrees()
+            expect[i] = scale*scale / (self.skyWcs.getPixelScale().asDegrees())**2
+        return expect
+
+    def testEvaluate(self):
+        expect = self._computeExpected(self.points)
+        result = np.zeros(len(self.points), dtype=float)
+        for i, point in enumerate(self.points):
+            result[i] = self.boundedField.evaluate(point)
+        self.assertFloatsAlmostEqual(expect, result)
+
+    def testEvaluateArray(self):
+        expect = self._computeExpected(self.points)
+        x = np.array([p.getX() for p in self.points])
+        y = np.array([p.getY() for p in self.points])
+        result = self.boundedField.evaluate(x, y)
+        self.assertFloatsAlmostEqual(expect, result)
+
+    def testEquality(self):
+        # not equal to a different type of BoundedField
+        other = lsst.afw.math.ChebyshevBoundedField(self.bbox, np.random.random((2, 2)))
+        self.assertNotEqual(self.boundedField, other)
+
+        # equal to something created with the same parameters.
+        other = PixelScaleBoundedField(self.bbox, self.skyWcs)
+        self.assertEqual(self.boundedField, other)
+
+        # not equal to something with different bbox.
+        newBox = lsst.geom.Box2I(self.bbox)
+        newBox.grow(10)
+        other = PixelScaleBoundedField(newBox, self.skyWcs)
+        self.assertNotEqual(self.boundedField, other)
+
+        # not equal to something with different wcs
+        crpix = self.skyWcs.getPixelOrigin()
+        crpix.scale(10)
+        newWcs = lsst.afw.geom.makeSkyWcs(crpix=crpix,
+                                          crval=self.skyWcs.getSkyOrigin(),
+                                          cdMatrix=self.skyWcs.getCdMatrix())
+        other = PixelScaleBoundedField(self.bbox, newWcs)
+        self.assertNotEqual(self.boundedField, other)
+
+    def testString(self):
+        # TODO: anything else to put in a str or repr?
+        self.assertIn("PixelScaleBoundedField(", repr(self.boundedField))
```

Comment:
I think this needs to be more complete.

Reply:
There's nothing else to test, given that there's no `str` implementation.

Reply:
I don't understand why we're testing a not-implemented method at all.

I also think an implementation of `str` that writes the pointer address of the internal `SkyWcs` is probably more useful than none at all (pity that `SkyWcs` doesn't provide any concise stringification), though I'm also not sure that is worth a test case.